src/shapes/Text/Text.ts

```diff
@@ -1640,23 +1638,23 @@ export class Text<
     styleObject?: TextStyleDeclaration,
     forMeasuring?: boolean
   ): string {
-    const style = styleObject || this,
-      family = this.fontFamily,
-      fontIsGeneric = Text.genericFonts.indexOf(family.toLowerCase()) > -1;
-    const fontFamily =
+    const style = styleObject || this;
+    const family = style.fontFamily || this.fontFamily;
+    const fontIsGeneric = Text.genericFonts.indexOf(family.toLowerCase()) > -1;
+    const formattedFontFamily =
       family === undefined ||
-      family.indexOf("'") > -1 ||
-      family.indexOf(',') > -1 ||
-      family.indexOf('"') > -1 ||
-      fontIsGeneric
-        ? style.fontFamily
+        family.indexOf("'") > -1 ||
+        family.indexOf(",") > -1 ||
+        family.indexOf('"') > -1 ||
+        fontIsGeneric
+        ? this.fontFamily
         : `"${style.fontFamily}"`;
```

Comment:
I thought the point was to use `family`

Reply:
Thank you @ShaMan123 for guiding me through.

The highlighted line (const family = style.fontFamily || this.fontFamily;) checks if the style object has a valid fontFamily property. If it does, family will be assigned the value of style.fontFamily. Otherwise, it will fall back to the this.fontFamily property.

I have run `npm run prettier:write`. Are there any other commands or tests that needs to be run?

Could you help against this 
<img width="349" alt="image" src="https://github.com/fabricjs/fabric.js/assets/69358136/c7a74368-7547-4912-8c39-cbc8a650b496">

Reply:
I have also added a unit test in the text/unit file and all tests are passing